### DISCUSSION TITLE
Running multiple indexes as one service in windows

### DISCUSSION THREAD

**Discussion by den-kip** [dadoonet/fscrawler#1145] [Q&A]
Writing a batch file for a single index job works fine. However, their is some challenges in adding more indexing jobs to the file as shown:

```
set JAVA_HOME=c:\Program Files\Java\jdk15.0.1
set FS_JAVA_OPTS=-Xmx2g -Xms2g

start "" /Elastic/fscrawler/bin/fscrawler.bat --config_dir C:\Documents\Elasctic\fscrawler job1 >> /Elastic/logs/fscrawler.log 2>&1
start "" /Elastic/fscrawler/bin/fscrawler.bat --config_dir C:\Documents\Elasctic\fscrawler job2 >> /Elastic/logs/fscrawler.log 2>&1
```

**Comment by dadoonet** [comment]
I don't understand the question or the bug report. Could you clarify what the problem is?

**Comment by dadoonet** [comment]
Just adding the link here as I can see that you also opened a discussion at https://stackoverflow.com/questions/67298541/run-several-indexes-as-a-service-using-fscrawler.

**Comment by dadoonet** [comment]
As you are running on Windows, I'd just open 2 different CMD and launch both separately.
